### PR TITLE
Send source maps to sentry

### PR DESCRIPTION
**Pull request by dawidgil**
feat for: https://energyweb.atlassian.net/browse/SWTCH-2194
Motivation: It is hard to detect what part of code is failing in Sentry, because code is minified and uglified. Adding source maps will fix it. 
Adding source maps won't affect security as switchboard code is open source project.

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -252,6 +252,14 @@
               ],
               "serviceWorker": true,
               "ngswConfigPath": "ngsw-config.json"
+            },
+            "development": {
```
Comment (jrhender) [inline angular.json:256]: I think it's quite confusing to have a `dev` and a `development` environment. Is one of them for local development? If so, could we use `local`?

Comment (dawidgil) [reply angular.json:256]: Yeah, you're right. It should be local. I will change that

Comment (dawidgil) [reply angular.json:256]: I see that nx is creating project with `development` and `develop` as a build for dev env. Maybe that would be a better option?

Comment (jrhender) [reply angular.json:256]: Sorry, I'm not following your comment here. Are you saying that nx doing both of these things:
1. creating project with `development`
2. creating `develop` as a build for dev env

Comment (dawidgil) [reply angular.json:256]: nx creates project with `develop` and `development` as configuration option. See this:
image
First one is for building for dev environment (minifying, uglifying, tree shaking etc.) like for prod, second is for starting application locally.

Comment (jrhender) [reply angular.json:256]: Ok, gotcha. Thanks for the screenshot

**Review by jrhender — APPROVED** [review]

**Pull request merged**